inputgraph/inputgraph.go

```diff
@@ -194,8 +194,20 @@ func (l *loader) handleFor(ctx context.Context, forStmt spec.ForStatement) error
 	return errors.Wrap(ErrUnableToDetermineHash, "for not supported")
 }
 
+func (l *loader) hashWaitStatement(waitStmt spec.WaitStatement) {
+	l.hasher.HashString("WAIT")
+	l.hasher.HashInt(len(waitStmt.Body))
+	l.hasher.HashJSONMarshalled(waitStmt.Args)
```

Comment:
we should set `v.SourceLocation = nil` here too

Reply:
Updated.